### PR TITLE
Suggested edits

### PR DESCRIPTION
**Pull request by fheathco**
All minor comments, mostly:

- making the EOL punctuation consistent in a file
- re-wording the phrase "the time" when the object was DATE type
- some word changes to make the phrase scan

### CODE REVIEW

**Comment by davidhyman** [comment]
Watch out, some (majority?) of those look like the result of the _codegen_ run against the API specifications!
The sources of those are the files listed at [this private internal link](https://confluence.arm.com/display/IoTBU/R1.2+Endpoints) - and the sources of _those_ may / may not be those actual files.

**Comment by thegecko** [comment]
@davidhyman None of the files edited are auto-generated, we are all good!

File: src/deviceDirectory/models/device.ts [hunk 1]
```diff
@@ -30,19 +30,19 @@ export class Device {
      */
     public readonly id: string;
     /**
-     * The owning IAM account ID
+     * The owning Ixxx Axxx Mxxxx (IAM) account ID [[[I think it would be helpful to expand this abbreviation]]]
```
Comment (thegecko) [inline src/deviceDirectory/models/device.ts]: `IAM` is `Identity and Access Management`. Could you update it, please?

File: src/update/models/firmwareImage.ts [hunk 2]
```diff
@@ -42,11 +42,11 @@ export class FirmwareImage {
      */
     public readonly datafileSize?: number;
     /**
-     * The time the object was created
+     * The timestamp when the object was created
      */
     public readonly createdAt: Date;
     /**
-     * The time the object was updated
+     * The timestamp when the object was updated
```
Comment (thegecko) [inline src/update/models/firmwareImage.ts]: There is trailing whitespace added to this line which has failed the build :(

File: src/update/models/firmwareManifest.ts [hunk 3]
```diff
@@ -46,11 +46,11 @@ export class FirmwareManifest {
      */
     public readonly deviceClass?: string;
     /**
-     * The time the object was created
+     * The timestamp when the object was created
      */
     public readonly createdAt: Date;
     /**
-     * The time the object was updated
+     * The timestamp the object was updated
```
Comment (thegecko) [inline src/update/models/firmwareManifest.ts]: There is trailing whitespace added to this line which has failed the build :(

**Pull request merged**